lib/screen/note_screen.dart

```diff
@@ -336,6 +344,37 @@ class _NoteScreen extends State<NoteScreen> with CommandHandler {
         ? Tuple2(notes.sublist(0, indexUnpinned), notes.sublist(indexUnpinned))
         : Tuple2(notes, []);
   }
+
+  void _deleteBook() async {
+    final uid = FirebaseAuth.instance.currentUser.uid;
+    final yes = await showDialog<bool>(
```

Comment:
`Yes` is quite poorly named. Give the fact that the user's response could also be "no". We can rename it "userResponse.